### PR TITLE
Test on PHP 8.2

### PR DESCRIPTION
**Pull request by ShawnCZek**

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -9,10 +9,11 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        php: [ 7.4, 8.0, 8.1 ]
+        php: [ 7.4, 8.0, 8.1, 8.2 ]
```
Comment (bensherred) [inline .github/workflows/run-tests.yml:12]: Should we drop support for 7.4 now?

Comment (ShawnCZek) [reply .github/workflows/run-tests.yml:12]: PHP 7.4 is still wildly used within the community. I think we should deprecate it together with 8.0 once it reaches its EOL, too. As PHP 8.1 brings enumerations that might be extremely useful for this library.

Comment (bensherred) [reply .github/workflows/run-tests.yml:12]: I’m happy to keep it for now if you would like to but I’d recommend drop it in time and then tag a major version. That way if a developer is using an older PHP version they can still use the package but without the latest features.

Comment (ShawnCZek) [reply .github/workflows/run-tests.yml:12]: We can do it in the next major version then.

**Pull request merged**